cmd/cmd.go

```diff
@@ -375,3 +396,26 @@ func initConfig() {
 		fs.Config.DataRateUnit = *dataRateUnit
 	}
 }
+
+func resolveExitCode(err error) {
+	if err == nil {
+		os.Exit(exitCodeSuccess)
+	}
+
```

Comment:
You should probably put `err = errors.Cause(err)` here so you unwrap the error to find the original error.

Reply:
Done!